docs/Acceptable-Formulae.md

```diff
@@ -38,7 +38,7 @@ There are exceptions:
 | emacs, vim         | [Too popular to move to dupes](https://github.com/Homebrew/homebrew/pull/21594#issuecomment-21968819) |
 | subversion         | Originally added for 10.5, but people want the latest version                                         |
 | libcurl            | Some formulae require a newer version than macOS provides                                             |
-| openssl            | macOS's openssl is deprecated & outdated.                                                             |
+| openssl            | macOS's openssl is deprecated & outdated                                                              |
```

Comment:
That space replaces the period so that the vertical bar at the end of the line still lines up with the others in the table.